app/src/main/res/layout-land/ongoing_topic_item.xml

```diff
@@ -14,68 +14,80 @@
   </data>
 
   <com.google.android.material.card.MaterialCardView
+    android:id="@+id/topic_container"
     android:layout_width="match_parent"
-    android:layout_height="92dp"
-    android:layout_marginStart="96dp"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="16dp"
     android:layout_marginTop="8dp"
```

Comment:
I missed this in last review. This should be `12dp` as the gap between two items is `24dp`

Reply:
I thought it was 16dp. I will correct it.

Reply:
Done.